Looks like there's a new but related failure:

```
======================================================================
ERROR: custom.icds_reports.tests.test_cas_data_export:TestLocationView.test_agg_awc_cas_data
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/test_cas_data_export.py", line 74, in test_agg_awc_cas_data
    csv_data = self._get_data_from_blobdb(indicator, state_id, month)
  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/test_cas_data_export.py", line 24, in _get_data_from_blobdb
    csv_file = io.TextIOWrapper(sync.get_file_from_blobdb(), encoding='utf-8')
  File "/mnt/commcare-hq-ro/corehq/blobs/util.py", line 52, in __getattr__
    return getattr(self._obj, name)
AttributeError: 'StreamingBody' object has no attribute 'readable'
```

--- a/custom/icds_reports/tests/test_cas_data_export.py
+++ b/custom/icds_reports/tests/test_cas_data_export.py
@@ -23,7 +23,11 @@ class TestLocationView(CSVTestCase):
 
     def _get_data_from_blobdb(self, indicator, state_id, month):
         sync, _ = get_cas_data_blob_file(indicator, state_id, month)
-        csv_file = io.TextIOWrapper(sync.get_file_from_blobdb(), encoding='utf-8')
+        with sync.get_file_from_blobdb() as streaming_body:
+            # Workaround botocore.response.StreamingBody has no 'readable' attribute
+            fileobj = io.BytesIO(streaming_body.read())
+        # TextIOWrapper uses universal newline mode by default
+        csv_file = io.TextIOWrapper(fileobj, encoding='utf-8')
         csv_data = list(csv.reader(csv_file))
         headers = csv_data[0]
         rows = csv_data[1:]

--- a/custom/icds_reports/tests/test_cas_data_export.py
+++ b/custom/icds_reports/tests/test_cas_data_export.py
@@ -10,10 +10,8 @@ import csv342 as csv
 
 from custom.icds_reports.tests import OUTPUT_PATH, CSVTestCase
 from six.moves import zip
-from unittest2 import skip
 
 
-@skip("This test is breaking the build on master and should be debugged on a branch.")
 class TestLocationView(CSVTestCase):
     always_include_columns = {'awc_id', 'case_id'}
     # indicator numbers:

